### ISSUE TITLE
Unfinished line or area lost when accidentally pressing save

### ISSUE THREAD

**Issue by Nekzuris** [openstreetmap/iD#10593]
### URL

https://www.openstreetmap.org/edit

### How to reproduce the issue?

Start drawing a line but don’t finish it, then click save, the line gets deleted.

### Screenshot(s) or anything else?

I understand we don’t want to save unfinished line to the OSM database, but I encountered this issue while drawing a long river; 
I made a mistake on the last point and wanted to hit `Ctrl + Z` to undo but miss clicked `Ctrl + S` and lost the entire line.

### Which deployed environments do you see the issue in?

Released version at openstreetmap.org/edit, Development version at ideditor.netlify.app

### What version numbers does this issue effect?

2.30.4

### Which browsers are you seeing this problem on?

Chrome, Firefox

**Comment by Suvendu-UI** [comment]
hey @Nekzuris can you send some pics related to this issue

**Comment by Nekzuris** [comment]
Pics won't help to show the issue, I should make a screen recording but I thought the issue was easy enough to reproduce @Suvendu-UI

**Comment by Adarsh-Dhar** [comment]
@Nekzuris would it make sense to save every unfinished line as draft, only push the finished line into the database

**Comment by Suvendu-UI** [comment]
@Nekzuris screen recording would be nice

**Comment by Nekzuris** [comment]
> would it make sense to save every unfinished line as draft

@Adarsh-Dhar No sure what you mean by draft, they should just be terminated to stay on the map. Objects with no tag already prevent push.

The issue is that the line gets deleted if I accidentally press `Ctrl + S` instead of `Ctrl + Z` while drawing it.

https://github.com/user-attachments/assets/56791b55-ae21-4820-9660-eb790a7a710a

@Suvendu-UI

**Comment by k-yle** [comment]
Hi, this has already been fixed in #10118

**Comment by Ravish990** [comment]
Is this issue has been closed ?? If not then I would love to fix it. Will you please assign the issue?? If yes then is there any other issue that I can solve . I love to contribute to the organization.

**Comment by Nekzuris** [comment]
> Hi, this has already been fixed in #10118

Actually no.

**Comment by Darshit42** [comment]
i think the best way to fix this is to add an end point to the line and end it where the last point was , or else we can show a message that there are still unfinished lines in the map and highlight it when saved

**Comment by Nekzuris** [comment]
Yeah that's also what I was thinking at first but the solution in #10118 to disable the shortcut could also work great. I don't think we need to show a message.

Are you updating your PR @k-yle ?

**Comment by Darshit42** [comment]
to disable the shortcut might be one way but i think many people are accustomed to ctrl + s , but we can also apply that too , ill try doing both of them , we can select which one's best

**Comment by Darshit42** [comment]
i dont think the bug has something to do with shortcuts , i tried recreating it again and after i refresh the page and select restore it shows the line, the bug is in refreshing the page , i think specifically with save button, should i raise the pr for this
